<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>708</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="font">
       <font>
        <family>MS Gothic</family>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>708</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="action_Open"/>
    <addaction name="action_Save"/>
    <addaction name="action_SaveAs"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <widget class="QMenu" name="menu_Character_Code">
     <property name="title">
      <string>&amp;Character Code</string>
     </property>
     <addaction name="action_UTF8"/>
     <addaction name="action_UTF8_with_BOM"/>
     <addaction name="action_EUCJP"/>
     <addaction name="action_ShiftJIS"/>
     <addaction name="action_UTF_16LE"/>
     <addaction name="action_UTF_16LE_with_BOM"/>
     <addaction name="action_UTF_16BE"/>
     <addaction name="action_UTF_16BE_with_BOM"/>
    </widget>
    <addaction name="action_Cut"/>
    <addaction name="action_Copy"/>
    <addaction name="action_Paste"/>
    <addaction name="separator"/>
    <addaction name="menu_Character_Code"/>
    <addaction name="separator"/>
    <addaction name="action_Wrap"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_New"/>
   <addaction name="action_Open"/>
   <addaction name="action_Save"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_New">
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_SaveAs">
   <property name="text">
    <string>Save &amp;As...</string>
   </property>
  </action>
  <action name="action_Wrap">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Wrap</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About...</string>
   </property>
  </action>
  <action name="action_UTF8">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;UTF8</string>
   </property>
  </action>
  <action name="action_UTF8_with_BOM">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>UTF8 with &amp;BOM</string>
   </property>
  </action>
  <action name="action_EUCJP">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;EUCJP</string>
   </property>
  </action>
  <action name="action_ShiftJIS">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;ShiftJIS</string>
   </property>
  </action>
  <action name="action_UTF_16LE">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>UTF-16LE</string>
   </property>
  </action>
  <action name="action_UTF_16LE_with_BOM">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>UTF-16LE with BOM</string>
   </property>
  </action>
  <action name="action_UTF_16BE">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>UTF-16BE</string>
   </property>
  </action>
  <action name="action_UTF_16BE_with_BOM">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>UTF-16BE with BOM</string>
   </property>
  </action>
  <action name="action_Cut">
   <property name="text">
    <string>Cu&amp;t</string>
   </property>
  </action>
  <action name="action_Copy">
   <property name="text">
    <string>&amp;Copy</string>
   </property>
  </action>
  <action name="action_Paste">
   <property name="text">
    <string>&amp;Paste</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
